episodes/collaborating-team.md

```diff
@@ -0,0 +1,256 @@
+- Review and provide feedback on contributions from collaborators using the GitHub interface.
+
+::::::::::::::::::::::::::::::::::::::::::::::::::
+
+:::::::::::::::::::::::::::::::::::::::: questions
+
+- How can I use GitHub to manage (report, assign, track, prioritise) tasks of a lesson development project?
+- What communication channels are effective for a lesson development team?
+- How can I use GitHub to manage contributions from collaborators?
+
+::::::::::::::::::::::::::::::::::::::::::::::::::
+
+
+In this episode we expand on how to use GitHub effectively among a group of known
+collaborators on an open source lesson development project, setting the stage for introducing tools and practices for
+working with newcomers. Here, we explore GitHub features to help you keep track of what needs doing on
+the lesson, manage contributions and prioritise work. Making decisions and managing your project as a whole will
+be covered in a follow-up episode on [project management](./project-management.html).
+Collaboration skills covered in this and follow-up episodes are transferable
+to any similar open source project you may be involved with.
+
+## Managing Issues
+
+**Issues** are GitHub's framework for managing issue/bug reports, feature requests, and lists of future work. They provide a single shared record of all the problems people have found with the lesson, and improvements that could be made, along with solutions and discussions around them. This helps the team to keep track of what they need to work on later, and reduces the chance of receiving redundant reports of issues you already know about.
+
+When you create an issue, you can add a range of details to it. An issue can be assigned to a specific team member - this can be a helpful way to know who is currently working to fix an issue or a way to assign responsibility to someone to deal with it. You can assign one or more labels to issues to classify or group them. Labels can also be given colours, allowing you to see at a glance from the Issues page the state of your project. The default labels available in GitHub include:
+
+- `bug` - indicates an unexpected problem or unintended behavior
+- `documentation` - indicates a need for improvements or additions to documentation
+- `duplicate` - indicates similar or already reported issues, pull requests, or discussions
+- `enhancement` - indicates new feature requests, or if they are created by a lesson maintainer, indicate planned new features
+- `good first issue` - indicates a good issue for first-time contributors
+- `help wanted` - indicates that a maintainer wants help on an issue or pull request
+- `invalid` - indicates that an issue, pull request, or discussion is no longer relevant
+- `question` - indicates that an issue, pull request, or discussion needs more information
+- `wontfix` - indicates that work won't continue on an issue, pull request, or discussion
```

Comment:
Maybe we should switch this to be a list of the Carpentries labels and then add how to add them into the repo?

Reply:
Or link out to that list? It is quite long...

Reply:
https://docs.carpentries.org/topic_folders/maintainers/github_labels.html

Reply:
Moving to own issue

Reply:
Now issue #260